### DISCUSSION TITLE
Set option from lazyvim option picker does not work

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#3001] [Q&A] [answered]
I recently found a cool lazyvim feature - that I can set option directly from option search. But it seems that it does nothing. But if I just use `set option=value` everything works fine.  Take a look:

https://github.com/LazyVim/LazyVim/assets/51330172/9536a5be-2414-4b08-9bcc-c546ca7ff803

Does it work how intended and I just don't understand something or is it a bug?

**Comment by dpetka2001** [comment, marked as answer]
From the description of `vim_options` it seems to allow the user to edit the value of the option in place. However, indeed I also can't seem to make it work. Probably a bug with the specific picker. I also tried without LazyVim with a minimal repro and it had the same behavior.
```lua
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not (vim.uv or vim.loop).fs_stat(lazypath) then
	vim.fn.system({
		"git",
		"clone",
		"--filter=blob:none",
		"https://github.com/folke/lazy.nvim.git",
		"--branch=stable", -- latest stable release
		lazypath,
	})
end
vim.opt.rtp:prepend(lazypath)

require("lazy").setup({
	{
		"nvim-telescope/telescope.nvim",
		dependencies = { "nvim-lua/plenary.nvim" },
		opts = {},
	},
})
```
